www/grpc/server_test.go

```diff
@@ -80,7 +80,7 @@ func setup(t *testing.T, conf *Config) *testData {
 	server := NewServer(
 		conf, mockState,
 		mockSync, mockNet,
-		mockConsMgr, wallet.NewWalletManager(mockState.Genesis().ChainType()),
+		mockConsMgr, wallet.NewWalletManager(mockState.Genesis().ChainType(), conf.WalletsDir),
```

Comment:
For test you can use `TempDirPath` function

Reply:
The way we construct temp dir is using `os.MkdirTemp`, and it'll return different value every time, so we cannot use `TempDirPath` here

Reply:
So, let's find a solution for that.
Look at the gRPC service as an object that has no knowledge about wallet. (path, chain-type, etc...)